src/Microsoft.Health.Fhir.Tests.Common/TestFiles/Normative/ValueSet.json

```diff
@@ -0,0 +1,45 @@
+{
+    "resourceType": "ValueSet",
+    "id": "cpt-all",
+    "url": "http://hl7.org/fhir/ValueSet/cpt-all",
+    "name": "All CPT codes",
+    "status": "active",
+    "date": "2015-03-12",
+    "publisher": "Health Level Seven International (Vocabulary)",
+    "description": "A value set that includes all CPT codes",
+    "useContext": [
+        {
+            "code": {
+                "system": "http://example.org/fhir/CodeSystem/use-contexts",
+                "code": "species"
+            },
+            "valueCodeableConcept": {
+                "coding": [
+                    {
+                        "system": "http://snomed.info/sct",
+                        "code": "337915000",
+                        "display": "Homo sapiens (organism)"
+                    }
+                ]
+            }
+        }
+    ],
+    "jurisdiction": [
+        {
+            "coding": [
+                {
+                    "system": "urn:iso:std:iso:3166",
+                    "code": "US"
+                }
+            ]
+        }
+    ],
+    "copyright": "CPT Copyright � 2014 American Medical Association. All rights reserved.",
```

Comment:
I don't think we want to be using a copyright for someone else in our test data. Should this file be edited so it isn't a direct copy?